pkg/server/sandbox_run.go

```diff
@@ -426,14 +427,29 @@ func toCNIDNS(dns *runtime.DNSConfig) *cni.DNS {
 	}
 }
 
-// selectPodIP select an ip from the ip list. It prefers ipv4 more than ipv6.
-func selectPodIP(ipConfigs []*cni.IPConfig) string {
+// selectPodIPs select an ip from the ip list. It prefers ipv4 more than ipv6
```

Comment:
TODO or ... Issue.. need a way to prefer ipv6 over ipv4. Or add preference to the ip order TODO below..

Reply:
Filed https://github.com/containerd/cri/issues/1278